index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Pigment</title>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css"
+    integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" crossorigin="anonymous" />
+
+
+  <link rel="stylesheet" href="/CSS/style.css" />
+  <link rel="stylesheet" href="/CSS/docs.css">
+  <link rel="stylesheet" href="/CSS/nav.css">
+</head>
+
+<body>
+
+  <div class="mobile-nav-button">
+
+    <div class="brand">
+      <h1><a href="/index.html">Pigment</a></h1>
+
+    </div>
+
+  </div>
+
+
+  <div class="desktop-main-nav">
+    <div class="brand">
+      <h1><a href="/index.html">Pigment</a></h1>
+    </div>
+
+    <div class="docs">
+      <h3><a href="/docs.html">Docs</a></h3>
+      <span>v.1.0.0</span>
+    </div>
+
+  </div>
+  <section class="hero">
+    <div class="hero-content">
+      <div class="hero-heading">
+        <h1>Design your project with pigment UI</h1>
+      </div>
+      <div>
+        <p>Pigment is a simple, customizable component library <br> for building quick and responsive sites. </p>
+      </div>
+      <div class="hero-btn">
+        <button class="btn-primary">
+          <a href="docs.html">
+            Get Started
+          </a>
+          </span>
+        </button>
+        <button class="btn-secondary">
+          <a href="https://github.com/kumaraswamy1/Component-library">

```

Comment:
update the GitHub url, this is giving a 404 page 
and you can also give `target="_blank"` here

Reply:
Done